src/statusBar.ts

```diff
@@ -7,16 +7,25 @@ import { ContainerManager, DevImageStatus } from "./containerManager";
 const imageToolTip = "Click to update image and respawn container.";
 
 export class StatusBarManager {
+  private buildUseCaseItem: vscode.StatusBarItem;
   private targetItem: vscode.StatusBarItem;
   private devImageItem: vscode.StatusBarItem;
 
-  constructor(target: string) {
+  constructor(target: string, useCase: string) {
     this.targetItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
     this.targetItem.tooltip = "Click to select another device.";
     this.targetItem.command = "selectTarget";
     this.targetItem.backgroundColor = new vscode.ThemeColor("statusBarItem.prominentBackground");
     this.updateTargetItem(target);
 
+    this.buildUseCaseItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
+    if (useCase !== "") {
+      this.buildUseCaseItem.tooltip = "Click to select another use case.";
```

Comment:
Tool tip should be more explicit : `Click to select a build use case`.

Reply:
Fixed